examples/Shim-Demo/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "labeling-tool-demo",
+  "version": "0.1.0",
+  "private": false,
+  "description": "Labeling tool demo project",
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.1.2",
+    "antd": "^4.16.1",
+    "dayjs": "1.10.7",
+    "labeling-tool": "^2.0.1",
+    "lodash": "^4.17.21",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-is": "^17.0.2",
+    "react-proptype-conditional-require": "1.0.4",
+    "react-scripts": "^5.0.1"
```

Comment:
ideally, here should be only CRA dependencies and labeling tool.
can u uninstall lodash, react-proptype-conditional-require, dayjs and antd? or it will not allow?

Reply:
@DrDiman yes, ideally, I started from only CRA dependencies and after that tried to make it work